README.rst

```diff
@@ -236,16 +236,16 @@ Async CSV Dict Reader
                     errors=kwargs.pop('errors', 'replace'),
                 ), **kwargs,
             )
+            self.line_num = 0
 
-        async def __aiter__(self):
-            header = await self.file_reader.readline()
-
-            if header:
```

Comment:
I can't seem to find a reason for this line, but I might be missing something :)

Was it just an optimisation of some sort?